### DISCUSSION TITLE
Update to AIR 33.1.1.731 flash.display.JPEGEncoderOptions can't be found

### DISCUSSION THREAD

**Discussion by twhamilton** [airsdk/Adobe-Runtime-Support#1514] [Q&A] [answered]
After updating our AIRSDK from AIR 33.1.1.633 to AIR 33.1.1.731 we are getting an error that flash.display.JPEGEncoderOptions could not be found. 
We overlay the SDKs over Flex SDK 4.6.0.

Can anyone else confirm this?   Any ideas  ?

**Comment by ajwfrost** [comment, marked as answer]
I've just double-checked, JPEGEncoderOptions is present as a definition within the airglobal.swc file. Not quite sure what could have gone wrong that would cause just this class to go missing although maybe if you're using Flash Builder then it's just bailing after finding this first problem...

My only suggestion would be to try this again, using a fresh version of the Flex SDK plus double-checking you have the correct variant of the AIR SDK for Flex developers...?

thanks

**Reply by twhamilton** [reply]
It must have been an issue with the overlay.  After re-doing that it seems everything is working as expected.
Thanks !

**Reply by ajwfrost** [reply]
Phew, thanks for confirming!
